README.md

```diff
@@ -55,11 +55,6 @@ Depending on what state the intermediate build products might be in, you may fir
 Licensed under MIT license ([LICENSE](LICENSE) or http://opensource.org/licenses/MIT).
 
 Downstream code forks should note that 'libzcashlc' (and thus XCFramework)
-depends on the 'orchard' crate, which is licensed under the [Bootstrap Open
-Source License](https://github.com/zcash/orchard/blob/main/LICENSE-BOSL).  A
-license exception is provided allowing some derived works that are linked or
-combined with the 'orchard' crate to be copied or distributed under the original
-licenses (in this case MIT), provided that the included portions of the
-'orchard' code remain subject to BOSL.  See
-https://github.com/zcash/orchard/blob/main/COPYING for details of which derived
-works can make use of this exception.
+depends on the 'orchard' crate, which is licensed under either of
+* Apache License, Version 2.0, (LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0)
+* MIT license (LICENSE-MIT or http://opensource.org/licenses/MIT)
```

Comment:
This entire paragraph can be deleted; `orchard` is now no different to our other dependencies.